assets/src/edit-story/components/panels/index.js

```diff
@@ -18,17 +18,13 @@
  * Internal dependencies
  */
 import { elementTypes } from '../../elements';
-import ColorPanel from './color';
-import StylePanel from './style';
 import PageBackgroundPanel from './pageBackground';
 import BackgroundSizePositionPanel from './backgroundSizePosition';
-import FontPanel from './font';
 import LinkPanel from './link';
 import MaskPanel from './mask';
 import LayerStylePanel from './layerStyle';
 import SizePositionPanel from './sizePosition';
-import TextStylePanel from './textStyle';
-import TextPanel from './text';
+import TextStylePanel from './textStyle/index';
```

Comment:
```suggestion
import TextStylePanel from './textStyle';
```

Reply:
For some reason, it gives an error like this locally. 😕 _Will test again just in case_

Reply:
Ah, because there's also a file called `textStyle.js`....why? 🤔

Reply:
Oh, it shouldn't be there, I guess merging re-created it at some point -- will fix.

Reply:
Hmm, still getting an error locally when not using `index`:  `Error: ENOENT: no such file or directory, ... `

Reply:
Works in browser though -- weird. Will deal with the local issue later and remove `index` from code.

Reply:
Removed in 5f9d25a